jsk_perception/node_scripts/ssd_object_detector.py

```diff
@@ -120,6 +124,16 @@ def image_cb(self, msg):
         bboxes, labels, scores = self.model.predict([img])
         bboxes, labels, scores = bboxes[0], labels[0], scores[0]
 
+        msg_indices = ClusterPointIndices(header=msg.header)
+        for bbox in bboxes:
+            H = img.shape[1]
+            W = img.shape[2]
```

Comment:
H and W can be assigned before for-loop instead of inside for-loop.

Reply:
H and W assigned before for-loop